vocode/streaming/synthesizer/eleven_labs_synthesizer.py

```diff
@@ -70,30 +121,34 @@ async def create_speech(
         create_speech_span = tracer.start_span(
             f"synthesizer.{SynthesizerType.ELEVEN_LABS.value.split('_', 1)[-1]}.create_total",
         )
-        async with self.aiohttp_session.request(
+
+        session = self.aiohttp_session
+
+        response = await session.request(
             "POST",
             url,
             json=body,
             headers=headers,
             timeout=aiohttp.ClientTimeout(total=15),
-        ) as response:
-            if not response.ok:
-                raise Exception(
-                    f"ElevenLabs API returned {response.status} status code"
-                )
+        )
+        if not response.ok:
+            raise Exception(f"ElevenLabs API returned {response.status} status code")
+        if self.experimental_streaming:
```

Comment:
i think there should also be an if branch on the url that we're hitting? otherwise the experimental streaming won't hit the 11Labs stream endpoint